ui/css/logviewer.css

```diff
@@ -9,17 +9,56 @@ body {
     bottom: 0;
     left: 0;
     min-width: 240px;
-    padding-top: 10px;
     display: flex;
     flex-flow: column;
 }
 
+/* Boostrap navbar overrides for logviewer */
+.navbar {
+    min-height: 41px; /* 40 for contained elements + lower bootstrap border */
+    border-top: 0; /* Shut off the top border */
+    border-radius: 0; /* Straight navbar */
+    margin-bottom: 10px; /* Breathing space for run-data */
+}
+
+.navbar-nav > li > a {
+    padding: 10px 15px;
+    font-size: 12px; /* Actionbar contents */
+}
+
+.navbar-nav > li > div {
+    padding: 10px 15px;
+    line-height: 20px;
+}
+
+.navbar-nav > li > span {
+    line-height: 20px;
+}
+
+.container-fluid {
+    padding-left: 5px; /* Align logo vertically to run-data table */
+}
+
 /* Suppress selected angular templates until they compile */
 [ng-cloak], [x-ng-cloak], .ng-cloak, .x-ng-cloak {
     display: none !important;
 }
 
-/*Log Viewer*/
+/* Log Viewer */
+
+#lv-logo {
+    padding: 10px 15px;
+    border: none;
+    background: #f8f8f8;
+}
+
+#lv-successful-steps {
+    font-size: 12px; /* Actionbar contents */
```

Comment:
I plan to remove this `/* Actionbar contents */` comment and also on L26 post-review. I was just providing them for review context in what they were prior.